test/qpy_compat/test_qpy.py

```diff
@@ -230,7 +230,8 @@ def generate_param_phase():
     return output_circuits
 
 
-def generate_single_clbit_condition_teleportation():
+def generate_single_clbit_condition_teleportation():  # pylint: disable=invalid-name
+    """Generate single clbit condition teleportation circuit."""
```

Comment:
I think Lev spotted that `pylint` didn't run on these files in tox because there's no `test/qpy_compat/__init__.py` file.

Reply:
I got pylint to complain about it locally for me. I think it's because of `lint-incr` which gets the files with changes and passes that directly to pylint

Reply:
Yeah, I bet Lev was using that as well.  CI doesn't moan, though, because the call `pylint test` searches for packages, so this file gets ignored.